src/services/form-validation.ts

```diff
@@ -623,6 +639,15 @@ export function generateValidationSchema(
             submission,
             formElementsConditionallyShown,
           )
+
+          const repeatableSetExecutedLookups =
+            executedLookups !== undefined &&
+            typeof executedLookups !== 'boolean' &&
+            !Array.isArray(executedLookups) &&
+            Array.isArray(executedLookups[formElement.name])
```

Comment:
Not sure if this would work but would likely be faster

```suggestion
            formElement.name in  executedLookups
```